yt_dlp/extractor/n1.py

```diff
@@ -81,16 +105,23 @@ def _real_extract(self, url):
         entries = []
         for video in videos:
             video_data = extract_attributes(video)
-            entries.append(self.url_result(
-                video_data.get('data-url'), ie=N1InfoAssetIE.ie_key(),
-                video_id=video_data.get('id'), video_title=title))
+            entries.append({
+                '_type': 'url_transparent',
+                'url': video_data.get('data-url'),
+                'id': video_data.get('id'),
+                'title': title,
+                'timestamp': timestamp,
+                'ie_key': N1InfoAssetIE.ie_key()})
 
-        youtube_videos = re.findall(r'(<iframe[^>]+>)', webpage)
-        for youtube_video in youtube_videos:
-            video_data = extract_attributes(youtube_video)
+        embedded_videos = re.findall(r'(<iframe[^>]+>)', webpage)
+        for embedded_video in embedded_videos:
+            video_data = extract_attributes(embedded_video)
             url = video_data.get('src')
             if url.startswith('https://www.youtube.com'):
                 entries.append(self.url_result(url, ie=YoutubeIE.ie_key()))
+            elif url.startswith('https://www.redditmedia.com'):
+                url = 'https://reddit.com' + url[27:]
+                entries.append(self.url_result(url, ie=RedditRIE.ie_key()))
```

Comment:
If all URLs extracted is sure to point to some video, you can just do this
```suggestion
            entries.append(self.url_result(url))
```
YoutubeDL code will try and figure out what extractor the URL should go to

Reply:
redditmedia.com url don't work merge #1090

Reply:
done